airbyte-webapp/src/packages/cloud/views/auth/SignupPage/SignupPage.module.scss

```diff
@@ -0,0 +1,11 @@
+@use "../../../../../scss/colors";
+
+.title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 39px;
+  width: 250px;
+  background: linear-gradient(to bottom, colors.$dark-blue-900 50%, colors.$blue-400 50%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
```

Comment:
:question: Could you clarify why we're using browser specific properties here? Are they really needed?

Reply:
Sure, let me explain the problem here we are trying to solve. Due to requirements [requirements](https://docs.google.com/document/d/16aT4ibOviQYendfR1LtCp3uPfDqi2tSOz-sqIfqx3PA/edit#), we need to have the ability to change the text above sign up form:

<img width="588" alt="image" src="https://user-images.githubusercontent.com/20929344/176517109-b0c4d38e-1211-483d-bd46-d41a4489add1.png">
<img width="1104" alt="image" src="https://user-images.githubusercontent.com/20929344/176517149-0608298b-0f42-4866-b16e-5df837d7afc7.png">

IMO We have two options to implement it:
1. Provide a divided message on 2 text blocks with defined colors.
2. Provide the whole message and control the colors via a linear gradient and width.

I've chosen the second one since it's more flexible.
So, to implement it, we need to provide the gradient as background and text, and just "clip" the text with that background color, here is the `background-clip` to rescue. But without a prefix it complains:

<img width="610" alt="image" src="https://user-images.githubusercontent.com/20929344/176521091-30868563-7056-4873-bbb3-8c49ec94c8a6.png">

Then we need to specify the fill color of characters in the text. `-webkit-text-fill-color` it's not standard, but it supports in all modern browsers as we can see:

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/20929344/176527322-dd640593-df68-4f60-92f7-582689a91045.png">

This is a price we need to pay for flexibility. If we'd have the static text, which wouldn't be overwritten - I'd choose the first option. 

WDYT?

Reply:
Okay thanks I think that clears it a bit better. Couldn't we instead just inject a "highlight" tag or similar into the message to mark which part is highlighted? This CSS solution imho is making quiet some assumptions about how many lines, etc.I think it would be nicer if we're having the message `Activate your <hl>14-day free trial</hl>`? I feel that might be the cleaner option (and would still allow during experimentation to overright which part of the message should be highlighted).